src/components/Search/SelectedRevisionsTable/types.tsx

```diff
@@ -0,0 +1,6 @@
+import { Alignment } from '../../../types/enums';
+
+export interface TableHeaderDetails {
+  title: string;
+  alignment: Alignment;
+}
```

Comment:
I think this should go in the `src/types` folder. also this does not need to be a `tsx` file